packages/material-ui/src/ButtonBase/ButtonBase.test.js

```diff
@@ -357,24 +364,8 @@ describe('<ButtonBase />', () => {
 
     it('should detect the keyboard', () => {
       assert.strictEqual(getState().focusVisible, false);
-      clock.tick(instance.focusVisibleCheckTime * instance.focusVisibleMaxCheckTimes);
-      assert.strictEqual(getState().focusVisible, true);
-    });
-
-    it('should ignore the keyboard after 1s', () => {
```

Comment:
We changed the duration at some point, entropy 👋

Reply:
There's alot of behavior that isn't tested right now in our codebase. But this is a fork so it is actually tested somewhere. Just need to make sure we watch for releases of the base.